src/circuit/README.md

```diff
@@ -101,15 +74,15 @@ Both for dialing and listening.
 
 ### Implementation rational
 
-This module is not a transport, however it implements `interface-transport` interface in order to allow circuit to be plugged with `libp2p-swarm`. The rational behind it is that, `libp2p-circuit` has a dial and listen flow, which fits nicely with other transports, moreover, it requires the _raw_ connection to be encrypted and muxed just as a regular transport's connection does. All in all, `interface-transport` ended up being the correct level of abstraction for circuit, as well as allowed us to reuse existing integration points in `libp2p-swarm` and `libp2p` without adding any ad-hoc logic. All parts of `interface-transport` are used, including `.getAddr` which returns a list of `/p2p-circuit` addresses that circuit is currently listening.
+This module is not a transport, however it implements `interface-transport` interface in order to allow circuit to be plugged with `libp2p`. The rational behind it is that, `libp2p-circuit` has a dial and listen flow, which fits nicely with other transports, moreover, it requires the _raw_ connection to be encrypted and muxed just as a regular transport's connection does. All in all, `interface-transport` ended up being the correct level of abstraction for circuit, as well as allowed us to reuse existing integration points in `libp2p` and `libp2p` without adding any ad-hoc logic. All parts of `interface-transport` are used, including `.getAddr` which returns a list of `/p2p-circuit` addresses that circuit is currently listening.
 
 ```
 libp2p                                                                                  libp2p-circuit (transport)
 +-------------------------------------------------+                                     +--------------------------+
 |        +---------------------------------+      |                                     |                          |
 |        |                                 |      |                                     |   +------------------+   |
 |        |                                 |      |  circuit-relay listens for the HOP  |   |                  |   |
-|        |           libp2p-swarm          <------------------------------------------------|  circuit-relay   |   |
+|        |              libp2p             <------------------------------------------------|  circuit-relay   |   |
 |        |                                 |      |  message to handle incomming relay  |   |                  |   |
 |        |                                 |      |  requests from other nodes          |   +------------------+   |
 |        +---------------------------------+      |                                     |                          |
```

Comment:
one last reference to libp2p-swarm below

Reply:
good catch!